packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasskey.tsx

```diff
@@ -0,0 +1,92 @@
+import type { ResetPasswordCodeFactor } from '@clerk/types';
+import React from 'react';
+
+import { useCoreSignIn } from '../../contexts';
+import { descriptors, Flex, Flow, Icon, localizationKeys } from '../../customizables';
+import { Card, Form, Header, IdentityPreview, useCardState } from '../../elements';
+import { Fingerprint } from '../../icons';
+import { useRouter } from '../../router/RouteContext';
+import { HavingTrouble } from './HavingTrouble';
+import { useHandleAuthenticateWithPasskey } from './shared';
+
+type SignInFactorOnePasswordProps = {
+  onShowAlternativeMethodsClick: React.MouseEventHandler | undefined;
+  onFactorPrepare: (f: ResetPasswordCodeFactor) => void;
+};
+
+export const SignInFactorOnePasskey = (props: SignInFactorOnePasswordProps) => {
+  const { onShowAlternativeMethodsClick } = props;
+  const card = useCardState();
+  const signIn = useCoreSignIn();
+  const { navigate } = useRouter();
+  const [showHavingTrouble, setShowHavingTrouble] = React.useState(false);
+  const toggleHavingTrouble = React.useCallback(() => setShowHavingTrouble(s => !s), [setShowHavingTrouble]);
+  const authenticateWithPasskey = useHandleAuthenticateWithPasskey();
+
+  const goBack = () => {
+    return navigate('../');
+  };
+
+  const handleSubmit: React.FormEventHandler = e => {
+    e.preventDefault();
+    return authenticateWithPasskey();
+  };
+
+  if (showHavingTrouble) {
+    return <HavingTrouble onBackLinkClick={toggleHavingTrouble} />;
+  }
+
+  return (
+    <Flow.Part part='password'>
+      <Card.Root>
+        <Card.Content>
+          <Header.Root showLogo>
+            <Icon
+              elementDescriptor={descriptors.passkeyIcon}
+              icon={Fingerprint}
+              sx={t => ({
+                color: t.colors.$neutralAlpha500,
+                marginInline: 'auto',
+                paddingBottom: t.sizes.$1,
+                width: t.sizes.$12,
+                height: t.sizes.$12,
+              })}
```

Comment:
@desiprisg @anagstef  Please let me know if we have new patterns that this code should align with.

In the designs this icon have a size of `48x48` which is not covered by the current Icon implementation and just adding an `xl` variant does not feel right as the `lg` represents a `20x20` so the difference would be quite large between them.

We may add a new `Header.Icon` component, but as long as this is used in only one place do we need them to ?

Reply:
Maybe the `Header.Icon` will make sense in order to have a proper descriptor instead of the `passkeyIcon` descriptor. WDYT ?

Reply:
It's ok to keep the width and height values.

Reply:
I'd say we don't add a `Header.Icon` yet. Let's keep the Icon as is for now. :)